### PR TITLE
Add a basic Mozilla theme

### PR DESCRIPTION
**Pull request by joewalker**

### CODE REVIEW

**Comment by bgrins** [comment]
@joewalker can you give instructions on how to build the repo into a website so I can test this?

**Comment by joewalker** [comment]
Honestly, the way I've been testing is to push to the gh-pages branch in my fork, which gives me http://incompleteness.me/firefox-browser-architecture/

I had a quick hack at jekyll and decided that `git push` was a lot easier. Maybe had I seen https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/ earlier or https://jekyllrb.com/docs/github-pages/ and https://jekyllrb.com/docs/quickstart/ then I might have stuck with the command line.

File: assets/css/style.scss [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+table {
+  margin-bottom: 1.25em;
+}
+
+a {
+  color: #006688;
+}
+
+a:hover, a:focus, a:active, a:visited:hover, a:visited:focus, a:visited:active {
+  color: #0088aa;
```
Comment (bgrins) [inline assets/css/style.scss:114]: Can we also do `font-weight: normal` here to prevent size changing when hovering links?

Comment (bgrins) [reply assets/css/style.scss:114]: (which appears to be coming from site.theme)

Comment (joewalker) [reply assets/css/style.scss:114]: Fixed in 2849291

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="{{ site.lang | default: "en-US" }}">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
```
Comment (bgrins) [inline _layouts/default.html]: I don't think we need to use chrome frame :)

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <meta name="viewport" content="width=device-width">
+    <!--[if lt IE 9]>
```
Comment (bgrins) [inline _layouts/default.html]: ..nor does this page use HTML5 tags

File: _layouts/default.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <div class="wrapper">
+      {{ content }}
+    </div>
+    <script src="{{ '/assets/js/scale.fix.js' | relative_url }}"></script>
```
Comment (bgrins) [inline _layouts/default.html:12]: This script doesn't exist - the reference can be removed

Comment (bgrins) [reply _layouts/default.html:12]: Nevermind, I didn't realize how the theme was working - looks like it does indeed exist at https://github.com/pages-themes/minimal/blob/master/assets/js/scale.fix.js

**Review by bgrins — APPROVED** [review]
The theming system is a little confusing, and it makes building the site locally difficult, but that's not because of this change. Ultimately I'd prefer to flatten all of the required assets from `jekyll-theme-minimal` and remove the reference from _config.yml but that's no reason to stop progress here.

File: _layouts/default.html (hunk 3 above)
Comment (joewalker) [reply _layouts/default.html]: Fixed in 3b668d4

File: _layouts/default.html (hunk 2 above)
Comment (joewalker) [reply _layouts/default.html]: Fixed in 3b668d4

**Pull request merged**